airfield/1.11/README.md

```diff
@@ -0,0 +1,94 @@
+# Airfield
+
+Airfield is an open source tool for the DC/OS ecosystem that enables teams to easily collaborate with shared [Apache Zeppelin](https://zeppelin.apache.org/) instances.
+
+## Requirements
+
+* DC/OS 1.11 or later
+* Marathon-LB
+* A wildcard DNS entry pointing at the loadbalancer. Each zeppelin instance will be available using a random name as a subdomain of your wildcard domain. As an example we will be using `*.zeppelin.mycorp`.
+* A Key-Value-Store to store the list of existing zeppelin instances. Currently supported are either [consul](https://www.consul.io/) or [etcd](https://coreos.com/etcd/). If you have neither installed we recommend [consul](https://github.com/MaibornWolff/dcos-consul) which is available in the [universe](https://universe.dcos.io/#/package/consul/version/latest). Check the [consul example](../../consul/1.11/README.md) in this repo for details on how to install it.
+* Enough available resources to run both Airfield and one Zeppelin instance (minimum: 3 cores, 10GB RAM).
+
+## Installation
+
+Airfield requires access to the Marathon API to manage zeppelin instances. If you are running DC/OS Enterprise you first need to create a serviceaccount for airfield:
+
+```bash
+dcos security org service-accounts keypair private-key.pem public-key.pem
```

Comment:
These are EE specific, did you test on OSS as well? Otherwise we can also make the Prereq EE...

Reply:
I tested it on OSS as well. That is why I added "If you are running DC/OS Enterprise you first need to create a serviceaccount" and provide options files for EE and OSS variants.